tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "compilerOptions": {
+    "target": "ESNext",
```

Comment:
I _think_ this is the right target. I don't think it affects the actual compilation, as that's up to Bable. It's just for typechecking, so it knows which language features are available